### PR TITLE
Finish Mobile Pull Request

### PR DESCRIPTION
**Pull request by alicemirigo92**
Add HTML
Make corrections where required
Style Header
Fix linters error

### CODE REVIEW

**Review by Felix45 — CHANGES_REQUESTED** [review]
Hi @alicemirigo92 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request :red_circle: 

You have submitted a project, that has failed linter checks. Please fix the failing stylelint linters and also provide a description of this pull request. Once you've done this please request for another code review :pray: 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by alicemirigo92** [comment]
[Felix45](https://github.com/Felix45) please assist..can we get to a zoom?

File: index.html [hunk 1]
```diff
@@ -21,15 +21,11 @@
         </div>
```
Comment (Yazino12) [inline index.html]: - [ ] Please make sure you wrap your hamburger menu in a `<nav>` tag 👍🏽 To make sure your code is semantically correct ✔.

image

Comment (Yazino12) [inline index.html]: - [ ] Make sure your page has the proper background image (decorations are always background images, not `img` tags). Example `background-image: url("./image/image1.png");` 🧱 _**If an image is not part of the content use CSS background-image**_

image

File: index.html [hunk 2]
```diff
@@ -21,15 +21,11 @@
...
         <section class="headline-section">
```
Comment (Yazino12) [inline index.html:18]: - **[OPTIONAL]** Please make sure your website has the title that best describes what your portfolio stands for 👊🏽.

image

**Review by Yazino12 — CHANGES_REQUESTED** [review]
## Hi @alicemirigo92 🖐🏾,

You've done well implementing some of the project requirements 🎉🎊🎉, but there are still some you need to work on to move to the next project 🔨.

### To highlight:
- Linters are passing ✔️
- Correct use of GitHub flow ✔️ 
- Descriptive Readme ✔

# STATUS:  REQUIRED CHANGES ♻♻♻ 
TRY AGAIN
_Check the comments under the review 👇🏽👇🏽👇🏽._

- [ ] From now on, try to write better commit messages. The sentences should be capitalized and in the imperative form. Example: "Modify readme file" and not "modify readme file" 👍🏽.

- [ ] kindly try to write descriptive and meaningful **PULL REQUEST** titles and summaries, something that explains the type of changes you made. 
 **_You can edit your summary by clicking on the three dots_** 👍🏽.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏 💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @Yazino12 when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (alicemirigo92) [reply index.html:18]: @Yazino12 done

Comment (alicemirigo92) [reply index.html:18]: @Yazino12 DONE

Comment (alicemirigo92) [reply index.html:18]: @Yazino12 Please review

**Review by Lizdev-05 — APPROVED** [review]
Hi  @alicemirigo92, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
approved

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by amejid — COMMENTED** [review]
Hi @alicemirigo92 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is supposed to be submitted for the "Portfolio: setup and mobile first" milestone. Kindly submit the appropriate project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**